adding-behavior/connecting-html-css-and-js.md

```diff
@@ -42,61 +35,64 @@ In our web projects, generally speaking, we can arrange our HTML, CSS, and JS fi
 
 ## Adding the Script Tag
 
-In order for our website to load HTML, we add a link to our JavaScript file in our HTML, similar to how we've included CSS.
+In order for our website to load HTML, we add a link to our JavaScript file in our HTML using the `<script>` tag, similar to how we've included CSS. Just like CSS links, we can include multiple `<script>`s in one page, and by default the scripts will be loaded in order.
 
 ```html
 <!-- index.html -->
 ...
-<body>
+<head>
   <!-- ... -->
 
-  <script src="src/index.js" type="text/javascript"></script>
-</body>
+  <script <optional async or defer attribute> src="src/index.js" type="text/javascript"></script>
+</head>
 ```
 
-Let's dive into this line, which we'll place at the end of the body, right before the closing `</body>` tag.
+Let's break down this line, which we'll place inside the head tag, usually right before the closing `</head>` tag.
 
 | <div style="min-width:200px;"> Piece of Code </div> | Notes                                                                              |
 | --------------------------------------------------- | ---------------------------------------------------------------------------------- |
 | `<script`                                           | Begins a `<script>` tag, a tag used to embed JavaScript                            |
+| <optional async or defer attribute>                 | We can optionally add the `async` or `defer` attributes to help us control how the script will load and execute |
```

Comment:
The table shows a blank spot for "Piece of Code" for the second row so we need backticks. I tested adding them and previewed that it would make the tag show up in the table.

```suggestion
| `<optional async or defer attribute>`                 | We can optionally add the `async` or `defer` attributes to help us control how the script will load and execute |
```

<img width="688" height="433" alt="Image" src="https://github.com/user-attachments/assets/acd73e10-135a-4aa2-9f36-ab8c1b51792f" />